src/docs/product/data-management-settings/server-side-sampling/getting-started.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Getting Started
+sidebar_order: 10
+description: "Learn how to get started with server-side sampling."
+---
+
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).
+
+</Note>
+
+<Note>
+
+To set up or configure server-side sampling, you need to be an Owner, Manager, or Admin of your Sentry organization.
+
+</Note>
+
+## Current Limitations
+
+Server-side sampling is currently only available for the following SDK's:
+
+- Python: 1.4.0 or later
+- JavaScript: 6.13.0 or later
+- Apple: 7.23.0 or later
+- Android: 6.2.1 or later
```

Comment:
Are these versions correct?
For example, the `source` was introduced in JavaScript [7.6.0](https://github.com/getsentry/sentry-javascript/releases/tag/7.6.0) and we need that.